src/bluefinClient.ts

```diff
@@ -2008,4 +2019,60 @@ export class BluefinClient {
       throw new Error(error);
     }
   };
+
+  /**
+   * @param to recipient wallet address
+   * @param balance SUI amount to transfer
+   * @returns Response Schema
+   * @description
+   * transfer sui tokens
+   * */
+  transferSuiBalance = async (
```

Comment:
Please make a single method `transferCoins` the method can expect the name of the coin as input and perform switch on the name

Reply:
done :rocket: